src/parse.jl

```diff
@@ -122,6 +122,20 @@ function _parse_colorant(desc::AbstractString)
         return RGBA{N0f8}(0,0,0,0)
     end
 
+    wo_spaces = replace(ldesc, r"(?<=[^ ]{3}) (?=[^ ]{3})" => "")
+    c = get(color_names, wo_spaces, nothing)
+    if c != nothing
```

Comment:
```suggestion
    if c !== nothing
```
In some cases the codegen is better. For example, check
```julia
julia> function checkiter1(iter)
           ret = iterate(iter)
           if ret == nothing
               println("nothing")
           else
               println("something")
           end
           nothing
       end

julia> function checkiter2(iter)
           ret = iterate(iter)
           if ret === nothing
               println("nothing")
           else
               println("something")
           end
           nothing
       end
```
with `@code_llvm checkiterX([])`.

Reply:
Understood. I'll fix it in the PR #371.